internal/check/valid_owner.go

```diff
@@ -48,6 +48,7 @@ type ValidOwner struct {
 	orgName              string
 	orgTeams             []*github.Team
 	orgRepoName          string
+	outsideCollaborators *map[string]struct{}
```

Comment:
```suggestion
	outsideCollaborators map[string]struct{}
```
FYI: We don't use pointers for map. Map is already a reference type, so it does not make sens here. To learn more, see:
https://go.dev/blog/maps